src/Vegetation/Roots.jl

```diff
@@ -243,61 +256,66 @@ function vc_integral_approx(
     b::FT,
     Kmax::FT,
 ) where {FT}
-    rhog_MPa = FT(0.0098)
+    ρg = FT(9800) # Pa/m
     u1 = a * exp(b * p1)
     u2 = a * exp(b * p2)
     num1 = log(u1 + FT(1))
     num2 = log(u2 + FT(1))
     c = Kmax * (a + FT(1)) / a
-    d = rhog_MPa * (z2 - z1)
-    flow_approx = -c / b * (num2 - num1) * (p2 - p1 + d) / (p2 - p1) # this is NaN if p2 = p1
-    A = c * d + flow_approx
-    B = -c * flow_approx / (b * A)
-    return u1, u2, A, B, flow_approx
+    d = ρg * (z2 - z1)
+    cond_area_flux_approx = -c / b * (num2 - num1) * (p2 - p1 + d) / (p2 - p1) # this is NaN if p2 = p1
+    A = c * d + cond_area_flux_approx
+    B = -c * cond_area_flux_approx / (b * A)
+    return u1, u2, A, B, cond_area_flux_approx
 end
 
 """
-    vc_integral(u1::FT, u2::FT, A::FT, B::FT, flow_approx::FT) where {FT}
+    vc_integral(u1::FT, u2::FT, A::FT, B::FT, cond_area_flux_approx::FT) where {FT}
 
-Computes the vc integral given the approximate flow.
+Computes the vc integral given the approximate cond_area_flux.
 """
-function vc_integral(u1::FT, u2::FT, A::FT, B::FT, flow_approx::FT) where {FT}
-    flow = B * log((u2 * A + flow_approx) / (u1 * A + flow_approx))
-    return flow
+function vc_integral(
+    u1::FT,
+    u2::FT,
+    A::FT,
+    B::FT,
+    cond_area_flux_approx::FT,
+) where {FT}
+    cond_area_flux =
+        B *
+        log((u2 * A + cond_area_flux_approx) / (u1 * A + cond_area_flux_approx))
+    return cond_area_flux
 end
 
 """
-    theta_to_p(theta::FT) where {FT}
+    θ_to_p(θ::FT) where {FT}
 
-Computes the volumetric water content (moles/moles) given pressure (p).
-Currently this is using appropriate vG parameters for loamy type soil.
-First the head (m) is computed, and then converted to a pressure in MPa.
+Computes the volumetric water content given pressure (p).
 """
-function theta_to_p(theta::FT) where {FT}
-    α = FT(5.0) # inverse meters
-    n = FT(2.0)
-    m = FT(0.5)
-    rhog_MPa = FT(0.0098) #MPa/m
-    p = -((theta^(-FT(1) / m) - FT(1)) * α^(-n))^(FT(1) / n) * rhog_MPa
+function θ_to_p(θ::FT) where {FT}
+    θ = min(θ, FT(1.0))
```

Comment:
We could use an exponentially growing function P(w) which might be more natural like we had discussed briefly